apps/server/src/utils/time.ts

```diff
@@ -31,7 +31,8 @@ export const isTimeString = (string) => {
  * @argument {string} date - excel string date
  * @returns {number} - time in milliseconds
  */
-export const dateToMillis = (date) => {
+
+export const dateToMillis = (date: Date): number => {
```

Comment:
@cpvalente did i type this correctly? jsdocs states date to be a string but on a string `getHours()` cant be called so i made it a `Date` type

Reply:
Yes, that is correct. The dateToMillis() seems to be called in parseExcelDate with a date object. good stuff!